src/main/java/Main.java

```diff
@@ -0,0 +1,42 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int n, t; //자동차 개수랑 경기횟수
+
+        ArrayList<String> names = saveNames(sc);
+
+        n = names.size(); //자동차 개수
+        System.out.println("시도할 회수는 몇회인가요?");
+        t = sc.nextInt(); //경기 횟수
+
+        Racing rc = new Racing();
+        rc.racing(n, t, names);
+        rc.winner();
+
+        print(rc);
+        
+        sc.close();
+    }
+
+    static ArrayList<String> saveNames(Scanner sc) {
+        System.out.println("경주할 자동차 이름을 입력하세요.");
+        String name_tmp = sc.next();
```

Comment:
```suggestion
        String nameTmp = sc.next();
```
일관되게 `CamelCase`로 명명해주세요